resources/views/navbar/items/dropdown-item.blade.php

```diff
@@ -0,0 +1,85 @@
+@props([
+    'name',
+    'description' => null,
+    'route'       => null,
+    'href'        => null,
+    'routeParams' => [],
+    'icon'        => null,
+    'disabled'    => false,
+    'external'    => false,
+    'tooltip'     => null,
+])
+
+@php ($isCurrent = $disabled === false && $route && url()->full() === route($route, $routeParams))
+
+<div class="flex">
+    <div @class([
+        'w-1 -mr-1 z-10',
+        'bg-theme-primary-600' => $isCurrent,
+    ])></div>
+
+    <a
+        @if ($disabled)
+            href="javascript:void(0)"
```

Comment:
🤔 with this line still clickable even if it does anything... maybe removing the `href` entirely?

Reply:
tweaked based on the slack conversation :+1: